src/trivyHelper.ts

```diff
@@ -4,12 +4,23 @@ import * as fs from 'fs';
 import * as toolCache from '@actions/tool-cache';
 import * as core from '@actions/core';
 import * as fileHelper from './fileHelper';
-const semver = require('semver');
+import * as table from 'table';
+import * as semver from 'semver';
+import * as utils from './utils';
 
 export const TRIVY_EXIT_CODE = 5;
 const stableTrivyVersion = "0.5.2";
 const trivyLatestReleaseUrl = "https://api.github.com/repos/aquasecurity/trivy/releases/latest";
 const trivyToolName = "trivy";
+const VULNERABILITIES = "Vulnerabilities";
```

Comment:
Prefix these constants with KEY, like `KEY_VULNERABILITIES`, `KEY_VULNERABILITY_ID` and so on?

Reply:
Done